com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -419,9 +448,27 @@ internal NetworkObject CreateLocalNetworkObject(bool isSceneObject, uint globalO
                     return null;
                 }
 
+                if (sceneObject.Header.HasTransform)
+                {
+                    if (worldPositionStays)
+                    {
+                        networkObject.transform.position = position;
+                        networkObject.transform.rotation = rotation;
+                    }
+                    else
+                    {
+                        networkObject.transform.localPosition = position;
+                        networkObject.transform.localRotation = rotation;
+                    }
+                }
+
+                // Go ahead and set network parenting properties
+                networkObject.SetNetworkParenting(isReparented, parentNetworkId, worldPositionStays);
```

Comment:
Does the order matter here? You're doing it differently than above...

Reply:
Ahh yeah I could move that.  Really, both `SetNetworkParenting` and setting the position and rotation need to occur before `NetworkObject.TrySetParent`.  Whether the position and rotation is set before `SetNetworkParenting`  or vice versa doesn't matter as much as they should be set prior to trying to set the parent.
However, I think keeping the same order of operations is cleaner and easier to understand.

Reply:
This was resolved when I consolidated this section of code.